.azure/gpu-tests-pytorch.yml

```diff
@@ -161,9 +161,8 @@ jobs:
       env:
         PL_USE_MOCKED_MNIST: "1"
         PL_RUN_CUDA_TESTS: "1"
-        PL_STANDALONE_TESTS_SOURCE: "pytorch_lightning"
+        PL_STANDALONE_TESTS_SOURCE: $(COVERAGE_SCOPE)
```

Comment:
@Borda This doesn't work: it's passing:

```$(' python -c ''\''n' = '"lightning"' ';' 'print(dict(fabric="lightning_fabric").get(n,' 'n))'\''' ')```

Reply:
shall be resolved in [0009239](https://github.com/Lightning-AI/lightning/pull/16705/commits/00092390cfe4156ced7277b040f799f6a859983b)